cli/install/template.go

```diff
@@ -401,14 +401,11 @@ spec:
           httpGet:
             path: /-/ready
             port: 9090
-          initialDelaySeconds: 30
-          timeoutSeconds: 30
```

Comment:
Given the [readiness probe defaults](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes), won't this only give the prometheus pod 20 seconds to start? Is that enough time in clusters that are resource-constrained? How would you feel about setting `failureThreshold: 7` on this probe, to give prometheus a full minute to start?

Reply:
I believe pods start as unready and then become ready as soon as they pass their readiness probe. Since the default `successThreshold` is 1, starting the readiness probe immediately will not delay a pod's readiness at startup time, nor will early readiness failures put the pod at risk of restart.